api/index.py

```diff
@@ -44,9 +44,19 @@ def do_GET(self):
 
             query = urllib.parse.parse_qs(raw_query)
         
-            if path not in ("/api", "/api/"):
+            if path not in ("/api", "/api/", "/api/analytics"):
                 raise NotFoundException
 
+            if path == "/api/analytics":
```

Comment:
Not sure if this is working on the Vercel deployment because I think routing works differently there than on a local server but this shouldn't stop the other objective of this PR from working.